src/resolvers/Query.ts

```diff
@@ -0,0 +1,22 @@
+import User from '../models/user.model'
+
+const Query:any ={
+    async findUser(parent: any, args: any, ctx: any, info: any): Promise<User> {
+        const user = await User.find(args.id)
+        return user 
+    },
+    async findUsersByName(parent: any, args: any, ctx: any, info: any): Promise<PaginatedUsersCompose> {
+        const user = await User.findByName(args.data.name, args.data.lastEvaluatedUser,args.data.pageSize)
+        return user 
+    },
+    async getUsers(parent: any, args: any, ctx: any, info: any): Promise<PaginatedUsersId>{
+       const users =  await User.getAll(args.lastEvaluatedId,args.pageSize)         
+       return users
+    },
+    async getAddressCoordinates(parent: any, args: any, ctx: any, info: any): Promise<String>{
+       const json =  await User.getCoordinateFromAddress(args.id)
+       return JSON.stringify(json)
```

Comment:
Do you see any drawbacks of returning a JSON string from this resolver instead of an object that is properly typed and defined in the schema?

Reply:
Yes it has it draw backs, because proper typing allow to clients of the api to know in advance what the response is going to content. I decided to use a JSON string for convenience